client/components/Home/Home.jsx

```diff
@@ -1,180 +1,170 @@
+                                hinders inclusion on the web for people with
+                                disabilities.
+                            </a>
+                        </p>
+                    </div>
+                    <div className="hero-video">
+                        <iframe
+                            src="https://player.vimeo.com/video/651279608?h=45aefd646f"
+                            width="640"
+                            height="360"
+                            frameBorder="0"
+                            allow="autoplay; fullscreen; picture-in-picture"
+                            allowFullScreen
+                            title="ARIA-AT Video"
+                        />
                     </div>
-                </div>
-                <div className="hero-illustration">
-                    <img
-                        src={heroImage}
-                        alt="An illustration of a computer where a test is being performed"
-                    />
                 </div>
             </section>
-            <section className="get-involved">
-                <div className="get-involved-container">
-                    <h2>Get Involved</h2>
-                    <ul className="resources">
-                        <li>
-                            <img src={iconJoinCommunity} alt="" />
-                            <a href="https://www.w3.org/community/aria-at/">
-                                Join the community group
-                            </a>
-                        </li>
+            <section className="improvements">
+                <div className="improvements-container">
+                    <h2>How are we improving interoperability?</h2>
+                    <ul className="improvement-list">
                         <li>
                             <img src={iconWriteTests} alt="" />
-                            <a href="https://github.com/w3c/aria-at/wiki/How-to-contribute-tests">
-                                Help write more tests
-                            </a>
+                            <h3>Proposing expectations for ATs</h3>
+                            <p>
+                                We have written initial drafts for more than a
+                                thousand tests that articulate expected screen
+                                reader behaviors for 40 examples of common web
+                                design patterns. View the{' '}
+                                <a href="https://github.com/w3c/aria-at/blob/master/metrics/coverage.md">
+                                    Test Writing Progress Report
+                                </a>{' '}
+                                and{' '}
+                                <a href="https://w3c.github.io/aria-at/build/">
+                                    view draft test plans
+                                </a>
+                                .
```

Comment:
Following internal discussions and confirmation, this period may cause unexpected behaviors in the screen reader output when immediately following the link. I can consistently reproduce an issue here in VoiceOver on Chrome while in reading mode.

Reply:
Let's change this to (link in bold):

> view the **draft test plans** preview.

Reply:
Fixed!